src/test/java/org/minions/devfund/reinaldo/movie/ActorTest.java

```diff
@@ -0,0 +1,76 @@
+package org.minions.devfund.reinaldo.movie;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by reinaldo on 25-07-18.
+ */
+public class ActorTest {
+
+
+
+    /**
+     * Calculate rating average.
+     */
+    @Test
+    public void testActorsMovieRatingAverageCalculation() {
+        final Actor actor = new Actor("Tom crush");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Tom crush" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)